src/main/java/Kelbot.java

```diff
@@ -4,22 +4,32 @@
     public static void main(String[] args) {
         Scanner sc = new Scanner(System.in);
         System.out.println("Hello! I'm Kelbot\n" + "What can I do for you?");
-        List<String> toDoList = new ArrayList<>();
+        List<Task> toDoList = new ArrayList<>();
         String input = sc.nextLine();
+        String[] commands = input.split(" ");
         while (true) {
             if (input.equals("bye")) {
                 break;
             } else if (input.equals("list")) {
-                // List out all the to-dos
+                // List out all the tasks
                 for (int i = 1; i <= toDoList.size(); i++) {
-                    System.out.println(i + ". " + toDoList.get(i - 1));
+                    System.out.println(i + "." + toDoList.get(i - 1));
                 }
+            } else if (commands[0].equals("done")) {
```

Comment:
perhaps make a variable for commands[0] = firstCommand for ease of reading

Reply:
Ah okay thanks!